### ISSUE TITLE
Set NuvlaBox location on the map not working

### ISSUE THREAD

**Issue by schaubl** [nuvla/ui#375]
A click on "save" button does nothing.

**Comment by schaubl** [comment]
When I click on the "save" button the following message appear on the JS console:
```
Error: No protocol method IAssociative.-assoc defined for type object: [object Object] nuvla-ui.js:3828:173
    Zd https://nuvla.io/ui/js/nuvla-ui.js:3828
    ye https://nuvla.io/ui/js/nuvla-ui.js:3837
    o https://nuvla.io/ui/js/nuvla-ui.js:3912
    hMa https://nuvla.io/ui/js/nuvla-ui.js:6402
    exports https://nuvla.io/ui/js/nuvla-ui.js:1550
    exports https://nuvla.io/ui/js/nuvla-ui.js:1555
    exports https://nuvla.io/ui/js/nuvla-ui.js:1550
    exports https://nuvla.io/ui/js/nuvla-ui.js:1557
    b https://nuvla.io/ui/js/nuvla-ui.js:2009
    l https://nuvla.io/ui/js/nuvla-ui.js:31
    f https://nuvla.io/ui/js/nuvla-ui.js:32
    k https://nuvla.io/ui/js/nuvla-ui.js:32
    h https://nuvla.io/ui/js/nuvla-ui.js:32
    ia https://nuvla.io/ui/js/nuvla-ui.js:52
    X https://nuvla.io/ui/js/nuvla-ui.js:51
    Ra https://nuvla.io/ui/js/nuvla-ui.js:52
    Ba https://nuvla.io/ui/js/nuvla-ui.js:54
    Qh https://nuvla.io/ui/js/nuvla-ui.js:255
    N https://nuvla.io/ui/js/nuvla-ui.js:37
    uc https://nuvla.io/ui/js/nuvla-ui.js:62
    $b https://nuvla.io/ui/js/nuvla-ui.js:61
    bind_applyFunctionN self-hosted:1042
    $b self-hosted:1005
    unstable_runWithPriority https://nuvla.io/ui/js/nuvla-ui.js:28
    zb https://nuvla.io/ui/js/nuvla-ui.js:80
    C https://nuvla.io/ui/js/nuvla-ui.js:255
    vc https://nuvla.io/ui/js/nuvla-ui.js:61
    bind_applyFunctionN self-hosted:1042
    vc self-hosted:1005
```

**Comment by 0xbase12** [comment]
Fixed in following commit 547b9eb
